### ISSUE TITLE
Lack of images in docker hub

### ISSUE THREAD

**Issue by fithisux** [apache/datafusion-ballista#1044]
**Describe the bug**
I am a windows user and I think the tools support Unix environments for building Docker images. E.g. cgroup

**To Reproduce**
-

**Expected behavior**
I expect a docker compose that sets up datafusion-ballista in my windows environment without having to build anything.

**Additional context**
I think it is beneficial for people becoming more aware of this tremendous project.

**Comment by tbar4** [comment]
@fithisux I don't think this is a bug and is instead a n "enhancement" or "user experience" request

**Comment by fithisux** [comment]
> @fithisux I don't think this is a bug and is instead a n "enhancement" or "user experience" request

How can I change it?
